app/models/reports/matview.rb

```diff
@@ -4,7 +4,29 @@ def self.refresh
       ActiveRecord::Base.transaction do
         ActiveRecord::Base.connection.execute("SET LOCAL TIME ZONE '#{Period::REPORTING_TIME_ZONE}'")
         Scenic.database.refresh_materialized_view(table_name, concurrently: true, cascade: false)
+        add_comments
```

Comment:
We don't really need to add comments with every refresh. I am looking for a way to abstract this away.